string   id
Position goalPos
---
Position[] path